### PR TITLE
Issue 688 Make pull and push urls clickable in the UI

### LINKED ISSUE #688: Make aggregate's URL clickable
#### Problem description or more improvement suggestion
I was wondering if we could do aggregate's URL clickable on the pull and push tab or at least make it possible to copy. 
It currently looks like: 
screenshot from 2018-11-28 11-26-39
screenshot from 2018-11-28 11-21-57
I think that hyperlink would be a very comfortable solution. It would also make an URL more visible in comparison to the other visible text because now it is not distinguishable nearly at all.
@yanokwa @ggalmazor what are your thoughts on that?

### PR DESCRIPTION
**Pull request by ggalmazor**
Closes #688

#### What has been done to verify that this works as intended?
Tested manually all possible sources on pull and push tabs.

#### Why is this the best possible solution? Were any other approaches considered?
I researched different options to make stuff clickable in Swing and I went for this one because it was the simplest way to get it done. Other alternatives involve more complex listeners and/or using custom-styled buttons instead of labels.

#### How does this change affect users? Describe intentional changes to behavior and behavior that could have accidentally been affected by code changes. In other words, what are the regression risks?

This PR makes clickable the URLs to the Aggregate sources/targets configured on the Pull and Push tabs. They will open the default browser.

#### Does this change require updates to documentation? If so, please file an issue at https://github.com/opendatakit/docs/issues/new and include the link below.
Nope.

### CODE REVIEW

File: src/org/opendatakit/briefcase/ui/reused/source/Source.java [hunk 1]
```diff
@@ -267,6 +279,23 @@ public String getDescription() {
...
+    private static void uncheckedBrowse(URL url) {
+      try {
+        getDesktop().browse(url.toURI());
+      } catch (URISyntaxException | IOException e) {
+        throw new BriefcaseException(e);
+      }
+    }
+
+    @Override
+    public void decorate(JLabel label) {
```
Comment (dcbriccetti) [inline src/org/opendatakit/briefcase/ui/reused/source/Source.java]: decorate is getting called twice, so clicking on the link creates two browser tabs.

Comment (ggalmazor) [reply src/org/opendatakit/briefcase/ui/reused/source/Source.java]: Good catch, thanks! I'll take a look

Comment (ggalmazor) [reply src/org/opendatakit/briefcase/ui/reused/source/Source.java]: Solved! Thanks again!

**Comment by kkrawczyk123** [comment]
Hhaha @ggalmazor you know me xd I couldn't wait long haha 
**Tested with success!!**
Verified on Ubuntu, MacOS and Windows and I like it very much :)

@opendatakit-bot unlabel "needs testing"
@opendatakit-bot label "behavior verified"

**Pull request merged**